doc/api/esm.md

```diff
@@ -616,8 +616,69 @@ console.log(five); // Logs `5`
 node b.mjs # works
 ```
 
-<i id="esm_experimental_loaders"></i>
+## HTTPS and HTTP imports
+
+> Stability: 1 - Experimental
+
+Importing network based modules using `https:` and `http:` is supported under
+the `--experimental-https-modules` flag. This allows web browser-like imports to
+work in Node.js with a few differences due to application stability and security
+concerns that are different when running in a privileged environment instead of
+a browser sandbox.
+
+### Imports are limited to HTTP/1
+
+Automatic protocol negotiation for HTTP/2 and HTTP/3 is not yet supported.
+
+### HTTP is limited to loopback addresses
+
+`http:` is vulnerable to man-in-the-middle attacks and is not allowed to be used
+for addresses outside of the IPv4 address 127.0.0.1 and the IPv6 address ::1.
```

Comment:
Per https://github.com/nodejs/node/pull/36328/commits/ae6f04742eaf8a8a19d3ab3356de33c8ffc4fa0a, the entire IPv4 loopback address block (from `127.0.0.1` to `127.255.255.255`) is now allowed. The documents here should be updated.